stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/message/list/background/MessageBackgroundFactoryImpl.kt

```diff
@@ -152,23 +152,23 @@ public open class MessageBackgroundFactoryImpl(private val style: MessageListIte
     override fun giphyAppearanceModel(context: Context): Drawable {
         return shapeAppearanceModel(
             context,
-            DEFAULT_CORNER_RADIUS,
-            SMALL_CARD_VIEW_CORNER_RADIUS,
+            defaultCornerRadius(),
+            smallCardViewCornerRadius(),
             isMine = true,
             isBottomPosition = true
         )
             .let(::MaterialShapeDrawable)
             .apply {
-                setTint(ContextCompat.getColor(context, MESSAGE_OTHER_USER_BACKGROUND))
+                setTint(ContextCompat.getColor(context, messageOtherUserBackground()))
             }
     }
 
     public companion object {
-        private val MESSAGE_OTHER_USER_BACKGROUND = R.color.stream_ui_white
-        private val MESSAGE_CURRENT_USER_BACKGROUND = R.color.stream_ui_grey_gainsboro
-        private val SMALL_CARD_VIEW_CORNER_RADIUS = 2.dpToPxPrecise()
+        private fun messageOtherUserBackground() = R.color.stream_ui_white
+        private fun messageCurrentUserBackground() = R.color.stream_ui_grey_gainsboro
+        private fun smallCardViewCornerRadius() = 2.dpToPxPrecise()
 
-        internal val DEFAULT_CORNER_RADIUS = 16.dpToPxPrecise()
+        internal fun defaultCornerRadius() = 16.dpToPxPrecise()
```

Comment:
Didn't get why do we need to turn those static primitives into functions
```java
   private static final int MESSAGE_OTHER_USER_BACKGROUND;
   private static final int MESSAGE_CURRENT_USER_BACKGROUND;
   private static final float SMALL_CARD_VIEW_CORNER_RADIUS;
   private static final float DEFAULT_CORNER_RADIUS;
```

Reply:
That's bugging me. `val` without const should be `private final`, but not static. Are you sure they'll become static values?

Reply:
Moved to class variable